packages/terra-hyperlink/CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ ChangeLog
 Unreleased
 ----------
 
+### Breaking Changes
+* Removed the following themeable variables:
+  * `--terra-hyperlink-external-after-background-image`
+  * `--terra-hyperlink-external-disabled-after-background-image`
+  * `--terra-hyperlink-image-after-background-image`
+  * `--terra-hyperlink-image-disabled-after-background-image`
+  * `--terra-hyperlink-video-after-background-image`
+  * `--terra-hyperlink-video-disabled-after-background-image`
+  * `--terra-hyperlink-audio-after-background-image`
+  * `--terra-hyperlink-document-after-background-image`
+  * `--terra-hyperlink-audio-disabled-after-background-image`
+  * `--terra-hyperlink-document-disabled-after-background-image`
+
+### Added
+* Added `--terra-hyperlink-visited-focus-color` themeable variable.
```

Comment:
Let's move these individual line-item details to the UpgradeGuide.md using the next minor version number (similar to [paginator](https://github.com/cerner/terra-core/blob/master/packages/terra-paginator/docs/UPGRADEGUIDE.md)) and only reference updates to themeable variables in one line in the changelog.

Reply:
I thought we do not want to add that information to Upgrade Guide anymore ([Matt's comment](https://github.com/cerner/terra-core/pull/2620#discussion_r332691707)). @mjhenkes @ryanthemanuel, where does this information go, going forward?

Reply:
We actually don't document this at all going forward.

Reply:
Gotcha! 😎I have [removed](https://github.com/cerner/terra-core/pull/2666/commits/fe59fc2d04b83b7e4879a24ef4caca81a66307c2) this piece.